cluster/ci/config/prow/openshift/prow-priority-class.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1beta1
+description: Used for prow pods that must run in the cluster, but can be
+  moved to another node if necessary.
+kind: PriorityClass
+metadata:
+  name: system-prow-critical
+value: 1999999000
```

Comment:
How did you arrive at this?

Reply:
`system-node-critical` is `2000001000`
`system-cluster-critical` is `2000000000` so I followed the patern here and put `2000000000 - 1000 = 1999999000`

Reply:
SGTM